### PR TITLE
Switch to nip.io from xip.io for default cluster up wildcard DNS

### LINKED ISSUE #13020: Use nip.io instead of xip.io to provide wildcard DNS for oc cluster up, etc
http://xip.io is flaky... Enter http://nip.io which in my few months of usage is pretty much rock solid.

### PR DESCRIPTION
**Pull request by jimmidyson**
Fixes #13020 if you're interested in doing that...

### CODE REVIEW

**Comment by mfojtik** [comment]
+1 I like nip.io more than xip.io

File: test/extended/builds/gitauth.go [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package builds
 
-// these tests are diabled because the xip.io dns hook was proving way too unreliable;
+// these tests are diabled because the nip.io dns hook was proving way too unreliable;
```
Comment (bparees) [inline test/extended/builds/gitauth.go]: s/diabled/disabled/

also is it still unreliable?

Comment (bparees) [inline test/extended/builds/gitauth.go]: (the latter is probably more a question for @csrwng )

**Comment by bparees** [comment]
one nit, otherwise lgtm.

File: test/extended/builds/gitauth.go (hunk 1 above)
Comment (csrwng) [inline test/extended/builds/gitauth.go]: We'd need to enable it and see. But I would create separate issue to look into that.

**Comment by smarterclayton** [comment]
Be sure to publicize this because people have scripted docs around it.

File: test/extended/builds/gitauth.go (hunk 1 above)
Comment (jimmidyson) [inline test/extended/builds/gitauth.go]: @csrwng How about enabling it in this PR and running tests a few times to see if it solves the issue? That would give more confidence that this change actually switches to a more stable wildcard DNS service?

Comment (jimmidyson) [inline test/extended/builds/gitauth.go]: Corrected old `diabled` typo btw

**Comment by e-minguez** [comment]
Wouldn't be nice to have our own cool domain like minishift.io? xip.io/nip.io are just dns servers with custom scripts... and as a bonus we can have some stats about who is using it :D

**Comment by jimmidyson** [comment]
Any more thoughts on this?

**Comment by csrwng** [comment]
@jimmidyson I haven't seen any strong objections from the mailing lists
LGTM

**Comment by bparees** [comment]
[merge]

**Pull request merged**